logrus/README.md

```diff
@@ -0,0 +1,32 @@
+# Go2sky with logrus (v1.8.1)
+
+## Installation
+
+```bash
+go get -u github.com/SkyAPM/go2sky-plugins/logrus
+```
+
+## Usage
+
+```go
+package main
+
+import (
+	"context"
+	"github.com/sirupsen/logrus"
+	logrusplugin "github.com/SkyAPM/go2sky-plugins/logrus"
+)
+
+func main() {
+	// init format with custom traceId key
+	logrus.SetFormatter(&logrusplugin.WrapFormat{&logrus.JSONFormatter{}, "TID"})
```

Comment:
TID? I think we should inject trace context. TID is an old thing.

Reply:
My fault, read this, https://github.com/SkyAPM/go2sky/issues/104#issuecomment-841640658